blocks/title/title.css

```diff
@@ -0,0 +1,125 @@
+.title {
+    /* The gradient here doesn't look great on desktop, better on mobile. Maybe we can create a slightly different one later */
+    background: var(--gradient);
+    width: 100vw;
+    margin-left: calc(-50vw + 50%);
+    padding: 26px;
+    border-top: 2px solid var(--blue);
+    border-bottom: 2px solid var(--blue);
+    position: relative;
+    margin-top: 80px;
+    margin-bottom: 40px;
+}
+
+.title h2 {
+    line-height: 1;
+    margin: 1px 0 0;
+    overflow-wrap: break-word;
```

Comment:
i think this might be worth setting in styles for all headings

Reply:
Added this to styles for headers :) Great feedback